sigma9.css

```diff
@@ -456,6 +458,83 @@ sup {
 	padding: 0 5px;
 }
 
+/* off-canvas */
+.close-menu {
+	display: none;
+}
```

Comment:
What is `.close-menu` here? It doesn't appear to be in a `@media` query.

Reply:
It's moved here from line 1334, randomly chucked between two media queries. This is a better place